File: 1374.generate-a-string-with-characters-that-have-odd-counts.py
#
# @lc app=leetcode id=1374 lang=python3
#
# [1374] Generate a String With Characters That Have Odd Counts
#

# @lc code=start
class Solution:
    def generateTheString(self, n: int) -> str:
        if n % 2:
            return 'a'*n
        else:
            return 'a'*(n-1)+'b'
# @lc code=end

File: 1374.generate-a-string-with-characters-that-have-odd-counts.cpp
/*
 * @lc app=leetcode id=1374 lang=cpp
 *
 * [1374] Generate a String With Characters That Have Odd Counts
 */
#include <string>
using namespace std;
// @lc code=start
class Solution
{
public:
    string generateTheString(int n)
    {
        if (n % 2)
        {
            return string(n, 'a');
        }
        else
        {
            return string(n - 1, 'a') + 'b';
        }
    }
};
// @lc code=end
